addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,647 @@
+				<label>Enable AD LightwaveRF messages</label>
+				<description>Enable AD LightwaveRF messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableMertik" type="boolean">
+				<context></context>
+				<label>Enable Mertik messages</label>
+				<description>Enable Mertik messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableVisonic" type="boolean">
+				<context></context>
+				<label>Enable Visonic messages</label>
+				<description>Enable Visonic messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableATI" type="boolean">
+				<context></context>
+				<label>Enable ATI messages</label>
+				<description>Enable ATI messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableOregonScientific" type="boolean">
+				<context></context>
+				<label>Enable Oregon Scientific messages</label>
+				<description>Enable Oregon Scientific messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableMeiantech" type="boolean">
+				<context></context>
+				<label>Enable Meiantech messages</label>
+				<description>Enable Meiantech messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableHomeEasyEU" type="boolean">
+				<context></context>
+				<label>Enable HomeEasy EU messages</label>
+				<description>Enable HomeEasy EU messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableAC" type="boolean">
+				<context></context>
+				<label>Enable AC messages</label>
+				<description>Enable AC messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableARC" type="boolean">
+				<context></context>
+				<label>Enable ARC messages</label>
+				<description>Enable ARC messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			<parameter name="enableX10" type="boolean">
+				<context></context>
+				<label>Enable X10 messages</label>
+				<description>Enable X10 messages to RFXCOM transceiver.</description>
+				<required>false</required>
+				<default>false</default>
+			</parameter>
+			
+		</config-description>
+	</bridge-type>
+	
+	<bridge-type id="RFXtrx433">
+		<label>RFXtrx433E USB 433.92MHz Transceiver</label>
+		<description>This is RFXCOM 433.92MHz transceiver bridge.</description>
+
+		<config-description>
+			<parameter name="bridgeId" type="text">
```

Comment:
No, only one bridge type is for manual configuration purposes, where user need to provide serial port. Rest of the bridges are used for discovered bridges by the JD2XX library, where identifier is RFXCOM controller serial number.